commit 5fbfd37f0fc5f4e7f25ded1d6812d8ba49fe579d
Author: Vitaly Kuznetsov <vkuznets@redhat.com>
Date:   Wed Dec 7 01:16:27 2016 -0800

    hv: don't reset hv_context.tsc_page on crash
    
    commit 56ef6718a1d8d77745033c5291e025ce18504159 upstream.
    
    It may happen that secondary CPUs are still alive and resetting
    hv_context.tsc_page will cause a consequent crash in read_hv_clock_tsc()
    as we don't check for it being not NULL there. It is safe as we're not
    freeing this page anyways.
    
    Signed-off-by: Vitaly Kuznetsov <vkuznets@redhat.com>
    Signed-off-by: K. Y. Srinivasan <kys@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1318_5fbfd37f0fc5f4e7f25ded1d6812d8ba49fe579d.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1318_5fbfd37f0fc5f4e7f25ded1d6812d8ba49fe579d.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
